javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/bootstrap/internal/CommonConfig.java

```diff
@@ -43,6 +44,8 @@ public static CommonConfig get() {
         config.getList("otel.instrumentation.http.capture-headers.server.response", emptyList());
     statementSanitizationEnabled =
         config.getBoolean("otel.instrumentation.common.db-statement-sanitizer.enabled", true);
+    emitExperimentalHttpClientMetrics =
+        config.getBoolean("otel.instrumentation.http.client.emit-experimental-metrics", false);
```

Comment:
I've also considered `...experimental-metrics.enabled`, `...capture-experimental-metrics` but honestly I wasn't really sure which one of them is better (or less bad)

Reply:
Seems fine this way to me.